h/browser/chrome/lib/sidebar-injector.js

```diff
@@ -288,6 +308,22 @@ function SidebarInjector(chromeTabs, dependencies) {
   function injectScript(tabId, path) {
     return executeScriptFn(tabId, {file: path});
   }
+
+  /**
+   * Inject configuration information for the Hypothesis application
+   * into the page as JSON data via a <meta> tag.
+   *
+   * A <meta> tag is used because that makes it available to JS content
+   * running in isolated worlds.
+   */
+  function injectConfig(tabId, config) {
```

Comment:
Is there any particular reason to use a `<meta>` tag rather than `<script class="js-hypothesis-settings" type="application/json">` as used in `app.html`? I ask only because it would seem sensible for there to be consistency between places we read settings data from the DOM.